modules/management-console-agent/src/main/java/org/apache/ignite/agent/processor/ClusterInfoProcessor.java

```diff
@@ -173,7 +154,5 @@ void sendClusterInfo(Event evt) {
         ctx.event().removeDiscoveryEventListener(this::sendTopologyUpdate, EVTS_DISCOVERY);
 
         ctx.event().removeLocalEventListener(this::sendClusterInfo, EVTS_CLUSTER_ACTIVATION);
-
```

Comment:
Missing `removeLocalEventListener` for EVTS_BASELINE_PARAMS_CHANGED, EVT_BASELINE_CHANGED

Reply:
Done.